src/test/java/org/broadinstitute/hellbender/tools/pon/PoNTestUtils.java

```diff
@@ -130,24 +113,29 @@ public static void assertEquivalentPoN(final File left, final File right) {
      * @param leftPoN never {@code null}
      * @param rightPoN never {@code null}
      */
-    public static void assertEquivalentPoN(final PoN leftPoN, final PoN rightPoN) {
+    public static void assertEquivalentPoN(final PCACoveragePoN leftPoN, final PCACoveragePoN rightPoN) {
         Utils.nonNull(leftPoN, "Left PoN is null.");
         Utils.nonNull(rightPoN, "Right PoN is null.");
-        Assert.assertEquals(leftPoN.getSampleNames(), rightPoN.getSampleNames());
-        Assert.assertEquals(new LinkedHashSet<>(leftPoN.getTargetNames()), new LinkedHashSet<>(rightPoN.getTargetNames()));
-        PoNTestUtils.assertEqualsMatrix(leftPoN.getTargetFactors(), rightPoN.getTargetFactors(), false);
+
+        Assert.assertEquals(leftPoN.getTargets(), rightPoN.getTargets());
+        Assert.assertEquals(leftPoN.getRawTargets(), rightPoN.getRawTargets());
+        Assert.assertEquals(leftPoN.getPanelTargets(), rightPoN.getPanelTargets());
+
+        PoNTestUtils.assertEqualsDoubleArrays(leftPoN.getTargetFactors(), rightPoN.getTargetFactors());
```

Comment:
Doesn't Assert.assertEquals(Object, Object) recognize if its arguments are `double[]`s and proceed accordingly?